.ci/.e2e-tests-daily.yaml

```diff
@@ -5,51 +5,51 @@ SUITES:
     scenarios:
```

Comment:
I think we want to include debian11 and ubuntu in the nightly builds. So I guess they should be added here too.

@jlind23 could you confirm?